migrate_reset.sh

```diff
@@ -0,0 +1,4 @@
+docker-compose up -d db
+export DATABASE_URL='postgresql://postgres:docker@localhost:5432/urefer?schema=public'
+cd ./backend
+npx prisma migrate reset --skip-seed # seeding will be done by run...?
```

Comment:
thoughts on have the migrate_reset file and if it should seed or not?

Reply:
if u didnt know migrate reset drops all the tables and creates empty tables and then applies all migrations

Reply:
@slustig210 I think seeding is reasonable 👍